indexer_test.go

```diff
@@ -24,7 +24,7 @@ func setup(t *testing.T) (*sql.DB, *elastic.Client) {
 	testDB, err := ioutil.ReadFile("testdb.sql")
 	assert.NoError(t, err)
 
-	db, err := sql.Open("postgres", "postgres://localhost/elastic_test?sslmode=disable")
+	db, err := sql.Open("postgres", "postgres://postgres@localhost/elastic_test?sslmode=disable")
```

Comment:
small issue, tests execute under a `travis` user which is fine when using supported travis DB services because we do get `travis` db user out-of-the-box, however, in other cases we either need to create a travis db user, or specify a user for the database connection

I've opted to specify a database user in the database connection string